velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,29 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseLaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double p) {
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(p)) {
+      result = std::numeric_limits<double>::quiet_NaN();
+    } else if (scale <= 0 || p < 0 || p > 1) {
+      result = std::numeric_limits<double>::quiet_NaN();
+      VELOX_USER_CHECK_GT(scale, 0, "scale must be greater than 0");
+      VELOX_USER_CHECK_GE(
```

Comment:
Please use the same error messages as the  java code.

Java code uses error "p must be in the interval [0,1]"